etc/gohan.yaml

```diff
@@ -67,4 +67,13 @@ extension:
   use:
   - gohanscript
  # - javascript
-  - go
\ No newline at end of file
+  - go
+
+  # default time limit in secs for running extension
+  timelimit: 10
+
+  # time limit overrides in secs for running particular extension in path
+  timelimits:
+  - path: /v1.0/store/orders
+    event: pre_*
+    timelimit: 1
```

Comment:
If it wouldn't be for the consistency with the existing field 'timelimit' I'd change it according to your suggestion. IMO, its better to copy the problematic naming in this case.